pages/documentation/grid/formats/psse.md

```diff
@@ -2,19 +2,130 @@
 layout: default
 ---
 
-# PSS/E
+# PSS®E
+[PSS®E software](https://new.siemens.com/global/en/products/energy/energy-automation-and-smart-grid/pss-software/pss-e.html) from Siemens provides analysis functions for power system networks in steady-state and dynamic conditions. PSS®E uses different types of files to exchange data about the network. One of them is the RAW file (power flow data file). A PSS®E RAW file contains a collection of unprocessed data that specifies a bus branch network model for the establishment of a power flow working case.
```

Comment:
We use in the web site Bus/Branch and not bus branch

Reply:
Done